File: csharp/DVSClient/DVSClient/server/address/format/RestApiAddressMetadataInfo.cs
using Newtonsoft.Json;

namespace DVSClient.Server.Address.Format
{
    public class RestApiAddressMetadataInfo
    {
        [JsonProperty("sources")]
        public IEnumerable<string>? Sources { get; set; }

        [JsonProperty("number_of_households")]
        public string? NumberOfHouseholds { get; set; }

        [JsonProperty("just_built_date")]
        public string? JustBuiltDate { get; set; }

        [JsonProperty("identifier")]
        public RestApiAddressMetadataInfoIdentifier? Identifier { get; set; }
    }
}

File: typescript/src/server/address/format/restApiAddressMetadataInfo.ts
import { RestApiAddressMetadataInfoIdentifier } from "./restApiAddressMetadataInfoIdentifier";

export type RestApiAddressMetadataInfo = {
    sources?: string[];
    number_of_households?: string;
    just_built_date?: string;
    identifier?: RestApiAddressMetadataInfoIdentifier;
};